e2e/__tests__/transform.test.ts

```diff
@@ -185,3 +185,25 @@ describe('transformer-config', () => {
     expect(stdout).toMatchSnapshot();
   });
 });
+
+describe('transformer caching', () => {
+  const dir = path.resolve(__dirname, '..', 'transform/cache');
+  const dirRegexp = new RegExp('^' + dir);
+
+  it('does not rerun transform within worker', () => {
+    // --no-cache because babel can cache stuff and result in false green
+    const {stdout} = runJest(dir, ['--no-cache', '-w=2']);
+
+    const loggedFiles = stdout.split('\n').map(line => {
+      expect(line).toMatch(dirRegexp);
```

Comment:
miiight need to use `slash` here for windows, actually. we'll see once CI runs

Reply:
I think yes

![image](https://user-images.githubusercontent.com/1404810/64003499-a46a8080-cb0c-11e9-8ff3-e0e255e56172.png)